src/v2/api/index.md

```diff
@@ -1178,10 +1178,10 @@ All lifecycle hooks automatically have their `this` context bound to the instanc
 
 ## Instance Methods / Lifecycle
 
-<h3 id="vm-mount">vm.$mount( [elementOrSelector] )</h3>
+<h3 id="vm-mount">vm.$mount( el, hydrating )</h3>
```

Comment:
Oh, indeed.
Search is broken...
I think try to merge your PR as quick fix.
However, you should be fix arguments of `$mount`.

`<h3 id="vm-mount">vm.$mount( [el, [hydrating] )</h3>`